src/arch/arm/armv/armv7-a/cache.c

```diff
@@ -10,11 +10,6 @@
 
 #include <arch/machine/hardware.h>
 
-static inline void invalidateByWSL(word_t wsl)
```

Comment:
`set_irq_active` just looks like an internal helper that was never used, it can go